x/oracle/types/keys.go

```diff
@@ -25,4 +26,13 @@ var (
 	// IDIndexCurrencyPairKeyPrefix is the key-prefix under which a currency-pair index.
 	// is stored.
 	IDIndexCurrencyPairKeyPrefix = collections.NewPrefix(3)
+
+	// NumRemovesKeyPrefix is the key-prefix under which the number of removed CPs is stored.
+	NumRemovesKeyPrefix = collections.NewPrefix(4)
+
+	// NumCPsKeyPrefix is the key-prefix under which the number CPs is stored.
+	NumCPsKeyPrefix = collections.NewPrefix(5)
+
+	// NumRemovesCodec is the collections.KeyCodec value used for the numRemoves value.
+	NumRemovesCodec = codec.KeyToValueCodec[uint64](codec.NewUint64Key[uint64]())
```

Comment:
We don't need a codec for `NumCPs`

Reply:
no - its the same codec for both.

I will rename to make this clearer